apps/mobile/src/components/widgets/collectibles/components/collectible-text.tsx

```diff
@@ -0,0 +1,45 @@
+// FIXME: Lifted from extension
```

Comment:
Should I move this code into the UI library alongside the extension code for displaying collectibles? 

I didn't setup `sanitize` from `'dompurify'` here yet as I was thinking I could install that once in the UI library instead and then load these components from there

Reply:
I think it makes to colocate code if the components are the same in both.

Reply:
Thanks. They are almost the same apart from some nuances ( e.g. `<Image` vs `<image`) 

Do you know where I could find ordinal collectibles of type `image` / `audio` ? No worries if not, I can ask support or try and source some.

When I test with the dev wallet all I could find was `html` / `text` which I have handled here. My code for `CollectibleImage` should handle image ordinals but I haven't tested it.

Reply:
Good question, I may have just hardcoded it to test ...think we've changed apis since then. Can you hardcode or buy from bestinslot.xyz directly?

Reply:
Sure, I will figure it out and test it properly. Thanks!

Reply:
I found samples of most inscription types apart from audio + video and added mocks. 

I can inscribe audio + video on ordinals bot if we think that's useful. 

For video the network fee is high so it's ~ $392 for what I tested. Audio can be cheaper ~ $122.02